### PR TITLE
TRUNK-7281 duplicate trunk check script in trunk_merge.sh

### PR DESCRIPTION
**Pull request by jennygzma**

### CODE REVIEW

File: trunk_merge.sh
[the remark this answers is not in the document]
Comment (sxlijin) [reply trunk_merge.sh:1]: save_annotations is unneeded and --github-annotate also will not work correctly in this context

[the remark this answers is not in the document]
Comment (sxlijin) [reply trunk_merge.sh:1]: `annotation_argument=""`

File: trunk_merge.sh [hunk 1]
```diff
@@ -23,6 +23,8 @@ echo "Detected merge queue commit, using HEAD^1 (${upstream}) as upstream and HE
 
 if [[ -n ${INPUT_CHECK_RUN_ID} ]]; then
   annotation_argument=--trunk-annotate=${INPUT_CHECK_RUN_ID}
+else
+  annotation_argument=--github-annotate
```
Comment (sxlijin) [inline trunk_merge.sh:27]:
```suggestion
  annotation_argument=""
```

not sure if github-annotate works on trunk_merge, we've never tested it afaik

Comment (jennygzma) [reply trunk_merge.sh]: oh i totally didn't catch that my bad, idk why i did --github-annotate

**Review by sxlijin — APPROVED** [review]

**Review by sxlijin — APPROVED** [review]

**Comment by jennygzma** [comment]
✨

**Pull request merged**